styles.css

```diff
@@ -55,23 +73,54 @@ ul {
   list-style-position: inside;
   width: 400px;
   padding-right: 50px;
+  padding-bottom: 12px;
 }
 
 ul li {
   padding-bottom: 8px;
 }
 
-.aboutMe {
+#aboutMe li {
```

Comment:
We always want to avoid to work with the ID-Selector, as we only use it in Java Script contexts. 
Please think of another Selector or simply add a classname to the listitems :)